.yamllint

```diff
@@ -1,9 +1,9 @@
+---
 extends: default
 
 ignore: |
-  .travis.yml
-  molecule/
-  tests/
+  /molecule/
```

Comment:
Instead we can just exclude tests under molecule directory `/molecule/**/tests/`